quantum/process_keycode/process_caps_word.c

```diff
@@ -83,7 +83,13 @@ bool process_caps_word(uint16_t keycode, keyrecord_t* record) {
 #endif // CAPS_WORD_IDLE_TIMEOUT > 0
 
     // From here on, we only take action on press events.
-    if (!record->event.pressed) {
+    // The exceptions are the combo results and the chord keys,
```

Comment:
At a glance, I thought that there was just kind of event generated from the combos implementation, where it calls `process_record()`, but I'm no expert. Are "combo result" and "chord key" synonymous terms, or is there a difference?

Reply:
I'm so sorry, I misclicked the "Resolve conversation" button... Is there any way to undo resolving?